manifest.php

```diff
@@ -25,7 +25,7 @@
     'label' => 'Proctoring',
     'description' => 'Proctoring for deliveries',
     'license' => 'GPL-2.0',
-    'version' => '3.6.16',
+    'version' => '3.6.17',
```

Comment:
I believe this is more an improvement so an intermediate version increase to 3.7 makes more sense, no ?